File: src/_02_StructuralPattern/_02_06_FlyWeight/java/after/Font.java
package _02_StructuralPattern._02_06_FlyWeight.java.after;

public class Font {
  
  private String fontFamily;
  private int fontSize;

  public Font(String fontFamily, int fontSize) {
    this.fontFamily = fontFamily;
    this.fontSize = fontSize;
  }

  public String getFontFamily() {
    return fontFamily;
  }

  public int getFontSize() {
    return fontSize;
  }
  
}

File: src/_02_StructuralPattern/_02_06_FlyWeight/Font.js
export default class Font {
  /** @type { string } */
  #fontFamily;

  /** @type { number } */
  #fontSize;

  /**
   * @param { string } fontFamily 
   * @param { number } fontSize 
   */
  constructor(fontFamily, fontSize) {
    this.#fontFamily = fontFamily;
    this.#fontSize = fontSize;
  }

  /** @returns { string } */
  getFontFamily() {
    return this.#fontFamily;
  }

  /** @return { number } */
  getFontSize() {
    return this.#fontSize;
  }
}